.idea/workspace.xml

```diff
@@ -0,0 +1,220 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="BranchesTreeState">
+    <expand>
+      <path>
```

Comment:
what's this?

Reply:
it's gone. i did a merge pull from master and i guess this somehow did an overwrite on my .gitignore file